lib/config.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
+
+const COLORS = require('./colors');
+
+
+let crowdinHelperJson;
+
+try {
+  crowdinHelperJson = JSON.parse(fs.readFileSync('crowdin-helper.json', 'utf8'));
+
+  const requiredConfigProperties = [
+    'projectIdentifier',
+    'projectKey',
+    'source',
+    'translation'
+  ];
+
+  let isSomeRequiredFieldsAbsent = false;
+
+  requiredConfigProperties.forEach(property => {
+    if (!crowdinHelperJson[property]) {
+      console.log(`Crowdin: ${COLORS.RED}Error: ${ property } is missing in crowdin-helper.json${COLORS.WHITE}`);
+      isSomeRequiredFieldsAbsent = true;
+    }
+  });
+
+  if (isSomeRequiredFieldsAbsent) {
+    throw new Error();
```

Comment:
Do we need this here?
Why can't we just raise an error at this line: https://github.com/yuriykuzin/crowdin-helper/pull/3/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9R23

Reply:
@sfate If we raise an error immediately, we'll tell about only one missed required prop. In current solution we're telling about all missed fields

Reply:
let user fix it one by one